example-mojaloop-backend/values.yaml

```diff
@@ -56,121 +43,25 @@ kafka:
     persistence:
       enabled: false
 
+    ## @param fullnameOverride String to fully override  template
+    ##
+    fullnameOverride: "zookeeper"
+
     ## @param affinity Affinity for pod assignment
     ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
     ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
     ##
     affinity: {}
 
-    ## @param nodeSelector Node labels for pod assignment
```

Comment:
Why removed?

Reply:
@mdebarros : I have defaulted affinity and pod selection back to the bitnami defaults , re-setting them here as we have is redundant and bloat